packages/react/src/components/Button/Button.props.ts

```diff
@@ -6,32 +8,55 @@ export interface ButtonProps {
   disabled?: boolean;
 
   /**
+   * If `true`, the button will occupy 100% of the width
    * @default false
    */
   fullWidth?: boolean;
 
   /**
-   * The text content of the button
+   * Check all available names in [Icon Library](https://ds.natura.design/28db352be/p/94367e-icon-library/b/6154b9)
```

Comment:
Acho que podia ter um textinho do que é a prop, tipo
"Set the icon to be rendered. Check all available..."